site/src/atomics/flex.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Flex Atomics
+description: Flex related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Flex Atomics
+
+Flex atomics are used to modify the layout of elements using flexbox and grid properties.
+
+## Usage
+
+### Flex-direction
```

Comment:
Been mulling this over for a bit, and I think this documentation page just has too much information on it (the best documentation source here not ironically MDN. For that reason, not sure that we should need to document the explicit behavior of all flexbox related values. Can we instead just put together a single example of composing a layout with flex atomics only?

Just as an example of something we could quickly put together with only flex atomics: https://codepen.io/wibjorn/pen/JjrvJaW

Not perfect, but maybe getting closer to a small-ish example of a working composable layout with flex helpers.

Reply:
Updated